src/app/zap-templates/zcl/data-model/chip/operational-credentials-cluster.xml

```diff
@@ -101,39 +131,46 @@ limitations under the License.
       <arg name="IPKValue" type="OCTET_STRING"/>
       <arg name="CaseAdminNode" type="NODE_ID"/>
       <arg name="AdminVendorId" type="INT16U"/>
+      <access op="invoke" privilege="administer"/>
     </command>
 
     <command source="client" code="0x07" name="UpdateNOC" response="NOCResponse" optional="false">
       <description>Sender is requesting to update the node operational certificates.</description>
       <arg name="NOCValue" type="OCTET_STRING"/>
       <arg name="ICACValue" type="OCTET_STRING" optional="true"/>
+      <access op="invoke" privilege="administer"/>
     </command>
 
     <command source="server" code="0x08" name="NOCResponse" optional="false">
       <description>Response to AddNOC or UpdateNOC commands.</description>
       <arg name="StatusCode" type="INT8U"/>
-      <arg name="FabricIndex" type="INT8U"/>
+      <arg name="FabricIndex" type="fabric_idx"/>
       <arg name="DebugText" type="CHAR_STRING"/>
```

Comment:
Need the field IDs for these args

Reply:
As none of the current cluster xml files include a fieldId element in the <arg/> attributes, just to make sure I understand the requirement,  would this format work?

```
    <command source="server" code="0x08" name="NOCResponse" optional="false">
      <description>Response to AddNOC or UpdateNOC commands.</description>
      <arg fieldId="0" name="StatusCode" type="INT8U"/>
      <arg fieldId="1" name="FabricIndex" type="fabric_idx"/>
      <arg fieldId="2" name="DebugText" type="CHAR_STRING"/>
    </command>
```

Reply:
None of the cluster xml files include a fieldId in the <arg/> attributes. To make sure I understand the requirement, is this format correct?

```
    <command source="server" code="0x08" name="NOCResponse" optional="false">
      <description>Response to AddNOC or UpdateNOC commands.</description>
      <arg fieldId="0" name="StatusCode" type="INT8U"/>
      <arg fieldId="1" name="FabricIndex" type="fabric_idx"/>
      <arg fieldId="2" name="DebugText" type="CHAR_STRING"/>
    </command>
```

Reply:
The last commit made the above assumption. If incorrect, we can rollback the changes and work on determining the right format.